src/actionCreator.js

```diff
@@ -1,17 +0,0 @@
-/* @flow strict-local */
-import type { Action, GlobalState, Dispatch } from './types';
-import { clearTypingNotification } from './typing/clearTypingNotification';
-import { EVENT_TYPING_START } from './actionConstants';
-
-export default (dispatch: Dispatch, actions: $ReadOnlyArray<Action>, state: GlobalState) => {
-  actions.forEach(action => {
-    switch (action.type) {
-      case EVENT_TYPING_START:
-        if (Object.keys(state.typing).length === 0) {
-          dispatch(clearTypingNotification());
-        }
-        break;
```

Comment:
So, it might be perfectly right to delete this, but it'll take a bit more reasoning than that :slightly_smiling_face: 

The commit message says:

    typing: Delete file 'actionCreator.js'.
    
    This file does not appear to do anything useful. So, delete it
    and remove references to it from all the places where it's called.

It sure looks like it's doing *something*, though! It's causing us to invoke `clearTypingNotification` in certain circumstances.

Is that not actually happening? Or, is it not useful to be doing that? In either case: why? :wink:

Reply:
Cool, sounds good. Thanks for drawing that connection!